backend/src/main/java/com/team4/backend/service/ReportService.java

```diff
@@ -0,0 +1,225 @@
+package com.team4.backend.service;
+
+import com.team4.backend.pdf.OffersPdf;
+import com.team4.backend.pdf.StudentsPdf;
+import org.springframework.stereotype.Service;
+import reactor.core.publisher.Mono;
+
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.util.*;
+
```

Comment:
No need to change, but this is too hard coded, honestly it will be hard to refactor it, and also generally we import constant from the properties file